include/industry_standard/spdm.h

```diff
@@ -13,8 +13,8 @@
 
 #pragma pack(1)
 
-/* 4 means SPDM spec 1.0, 1.1, 1.2, 1.3 */
-#define SPDM_MAX_VERSION_COUNT 4
+/* Covers SPDM specifications from 1.0 to 1.4. */
+#define SPDM_MAX_VERSION_COUNT 5
```

Comment:
Wait.

That will cause uninitialized data in https://github.com/DMTF/libspdm/blob/main/library/spdm_common_lib/libspdm_com_context_data.c#L3281-L3289

This patch may be fine. But it is NOT enough.

Reply:
NOTE: We already have `#define LIBSPDM_MAX_VERSION_COUNT 5` at https://github.com/DMTF/libspdm/blob/main/include/library/spdm_lib_config.h#L168.

I think we can handle the peer with 1.4.